### ISSUE TITLE
FreeBSD build breaks

### ISSUE THREAD

**Issue by aviumcaravan** [trofi/ski#4]
# SPARE_USRSPACE

* according to the commit history, on December 6th 2019 SPARE_USRSPACE got removed due to being unused legacy API, making the FreeBSD version no longer possible to build.

* proposed solution: defining it in ``src/freebsd/osload.c`` as 4096 (that's the value it's supposed to be) or replacing the call instance with an integer value.

# cannot build ``src/freebsd/signal.c``

* at line 45, position 7 of ``src/freebsd/signal.h`` the ``si_band`` macro is called causing the two issues in the picture below:

![screenshot showing build errors related to the macro](https://github.com/trofi/ski/assets/59111798/308af5ab-078c-467f-8304-27a29490c3a1)

* proposed solution: ...no idea, to be perfectly honest. maybe someone smarter than me can figure this out.

**Comment by trofi** [comment]
Yeah, i did quite a bit of cleanup for `linux` without too much focus on any other ports. I did not intend to break the other ports, thus there is still a chance `FreeBSD` could be amended without too much trouble.

I'll try to get a FreeBSD VM and explore a bit (until someone else sends a patch faster). I might get back to you if I fail to reproduce the failure.
